presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -487,6 +487,26 @@ public void createPartitionedTableAs()
         }
     }
 
+    @Test
+    public void testShowProperties()
+    {
+        String tableName = "test_show_properties";
```

Comment:
I know some other tests use a variable for the name, but I find it easier to read if we use the name directly in the queries. Also, drop the table at the end for cleanup.

Reply:
done